ui/src/admin/components/DatabaseTable.js

```diff
@@ -43,13 +49,21 @@ const DatabaseTable = ({
         isAddRPDisabled={!!database.retentionPolicies.some(rp => rp.isNew)}
       />
       <div className="db-manager-table">
-        <table className="table v-center admin-table">
+        <table className="table v-center table-highlight">
```

Comment:
this table has no multi-select dropdowns, and that's all `.admin-table` styles anymore